Rakefile

```diff
@@ -19,4 +19,4 @@ rescue LoadError
 end
 
 Rake::Task[:default].clear if Rake::Task.task_defined?(:default)
-task default: %i[rubocop spec pact:verify]
+task default: %i[rubocop spec pact:verify build_schemas]
```

Comment:
Do we want to do this every time a branch of Publishing API is built, as I'd imagine it will slow down the builds quite a bit?  I wonder if there's a way we can do a diff to see if any of the schemas have changed compared to the main branch?

Reply:
Good point. I've noted this on [the card](https://trello.com/c/iZup32bI/362-only-run-schemas-related-tests-and-build-steps-in-publishing-api-when-changes-are-made) I've got (to follow the current work) to look at speeding up publishing api's build post the content schemas merge